packages/types/src/index.ts

```diff
@@ -401,3 +401,22 @@ export interface CallUserExternal {
   display_name: string;
   avatar_url: string;
 }
+
+export interface SlashCommand extends Record<string, any> {
```

Comment:
```suggestion
export interface SlashCommand extends Record<string, unknown> {
```
Unknown may be better, but this is just keeping it the same as https://github.com/slackapi/bolt-js/blob/22f938e53ccef59e5a4acdd7b4d2a271430dd957/src/types/command/index.ts